### PR TITLE
Clean Up UI

### LINKED ISSUE #189: Update UI.png

### LINKED ISSUE #197: Clean Up UI
Last checks to finalize UI
- [x] Add children statement somewhere that keeps readding the ui elements into the hbox
- [x] Make AID PID label white
- [x] Others (if necessary)

### PR DESCRIPTION
**Pull request by jayjay19630**
Resolves #197. Resolves #189.

Minor improvements in UI 

Ui

### CODE REVIEW

**Comment by zaidansani** [comment]
Do you think it would be a good idea to have word wrap on the status bar instead of a scroll?

**Comment by choiwab** [comment]
All the changes lgtm!
One suggestion: should we make the AID box a different color (e.g. blue)?

File: src/main/resources/view/PersonListCard.fxml [hunk 1]
```diff
@@ -15,17 +15,17 @@
...
+        <Label fx:id="id" styleClass="cell_big_name_label">
           <minWidth>
             <Region fx:constant="USE_PREF_SIZE" />
           </minWidth>
         </Label>
-        <Label fx:id="name" styleClass="cell_big_label" text="\$first" />
+        <Label fx:id="name" styleClass="cell_big_name_label" text="\$first" />
         <Region HBox.hgrow="ALWAYS" />
         <Label styleClass="cell_id_prefix_label" text="PID:" />
         <Label fx:id="personId" styleClass="cell_id_label" text="\$personId" />
```
Comment (choiwab) [inline src/main/resources/view/PersonListCard.fxml:31]: Thank you for fixing the UI bug for the clearing command!

**Comment by jayjay19630** [comment]
> All the changes lgtm! One suggestion: should we make the AID box a different color (e.g. orange, green)?

After experimenting a bit, I think it's best if we use the consistent DocTrack color scheme for the IDs. Adding more colors could possibly make it more distracting. What do you think?

**Comment by jayjay19630** [comment]
> Do you think it would be a good idea to have word wrap on the status bar instead of a scroll?

Can I clarify with you what scroll you're referring to? Currently the status bar already truncates the string if the width of the window is too small.

Screenshot 2024-11-04 at 9 20 07 PM

Thanks in advance!

**Comment by zaidansani** [comment]
> Can I clarify with you what scroll you're referring to?

image

This one!

**Comment by zaidansani** [comment]
> After experimenting a bit, I think it's best if we use the consistent DocTrack color scheme for the IDs. Adding more colors could possibly make it more distracting. What do you think?

Came to a similar conclusion, I think since DocTrack itself only uses two colours, it might be a bit too drastic for a change

**Comment by jayjay19630** [comment]
OK, the word wrapping should be fixed now!

**Comment by zaidansani** [comment]
I think it'd be a good idea to change the `Ui.png` here too!

**Comment by jayjay19630** [comment]
> I think it'd be a good idea to change the `Ui.png` here too!

Okay, I've changed the Ui.png here as well!

**Comment by choiwab** [comment]
LGTM! Thank you for fixing the clear command UI bug

**Pull request merged**